README.md

```diff
@@ -31,12 +31,12 @@ brew(gui(App()))
 
 The code above will result in the following application:
 
-<img alt="Counter Application" src="docs/assets/introduction.png" width="228px">
+<img alt="Counter Application" src="../docs/assets/introduction.png" width="228px">
```

Comment:
This file still needs to work as the README of the GitHub Repository, currently the images are not displayed correctly there.

Reply:
I may not be seeing what you are seeing there.
The 2 places that need to work are:
https://github.com/PhilippMDoerner/owlkettle/tree/feature/nimibook-branch2
and
https://philippmdoerner.github.io/owlkettle/examples/README.html

Which render to me as:
![image](https://user-images.githubusercontent.com/65814317/221053671-d4ded0c9-46f1-499b-9ee2-f102898ae44d.png)
and
![image](https://user-images.githubusercontent.com/65814317/221053701-ea4a7615-c20f-4d5b-80ee-2b6178d8620d.png)

What am I missing?

Reply:
This file is the main README of the repository, not the examples/README.md file.

Reply:
Ohhhhh yeah I broke those links, one sec, that should be an easy fix

Reply:
Version with fixed links is now deployed for viewing, I'll mark this as resolved